doc/CMakeLists.txt

```diff
@@ -17,6 +17,24 @@ IF(DOXYGEN_FOUND)
         SET(DOXYGEN_HTMLHELP NO)
     ENDIF(HTML_HELP_COMPILER)
 
+    # -----------------------------------------------------------------------------
+    # Fetch doxygen stylesheet from https://github.com/jothepro/doxygen-awesome-css
+    # -----------------------------------------------------------------------------
+    SET(AWESOME_CSS_URL "https://raw.githubusercontent.com/jothepro/doxygen-awesome-css/main/doxygen-awesome.css")
+    SET(AWESOME_CSS_DESTINATION "${DOC_INPUT_DIR}/doxygen-awesome.css")
+    FILE(DOWNLOAD ${AWESOME_CSS_URL} ${AWESOME_CSS_DESTINATION}
+        STATUS DOWNLOAD_STATUS
+        LOG DOWNLOAD_LOG
+    )
```

Comment:
I wonder if we should just copy a static version of this into Cyclus?  On one hand, it could become stale, but on the other hand, it wouldn't update with no notice and suddenly change our docs.

I could go either way, but just looking for a brief discussion....

Reply:
I think thats a good strategy.  Fetching it adds complexity that probably isn't necessary for something like css.  I will update this